Prelude.playground/Pages/06-Traits.xcplaygroundpage/Contents.swift

```diff
@@ -53,6 +55,6 @@ button
   |> buttonStyle
 button.sizeToFit()
 
-let frame = parent.view.frame
-XCPlaygroundPage.currentPage.liveView = parent
-parent.view.frame = frame
+PlaygroundPage.current.liveView = parent.view
+
+child.view.addSubview(button)
```

Comment:
Adding to view hierarchy seems safer after the view has been assigned to `liveView`.